scala/lambdas/preingest-tdr-package-builder/src/main/scala/uk/gov/nationalarchives/preingesttdrpackagebuilder/Lambda.scala

```diff
@@ -234,9 +235,17 @@ class Lambda extends LambdaRunner[Input, Output, Config, Dependencies]:
       .map(_ => new Output(input.batchId, input.groupId, URI.create(s"s3://${config.rawCacheBucket}/${input.batchId}/metadata.json"), input.retryCount, ""))
   }
 
+  private def truncate(s: String) = {
```

Comment:
Since we're limiting it to 100, it makes this method quite specific, should we allow the char limit to be passed in maybe with a default of 100 or rename it `truncateTo100Chars`?

Reply:
You don't need to rename it, you can see it's truncating to 100 because there's 100 in the method and we're not going to want to truncate anything else to a different number of characters. If we do end up changing 100, it's only in one place.